### ISSUE TITLE
Overrides places where to jump when re-opening a file

### ISSUE THREAD

**Issue by mrfuxi** [titoBouzout/BufferScroll#28]
When I have a file closed (that I already viewed) and want to open it with command line as:
```
subl path/to/file.py:10
```
or when jumping from diff (hit Enter on the beginning on the line) then file is opened and scrolled to the last position remembered by BufferScroll rather then to the line I'm after

**Comment by titoBouzout** [comment]
Yes, the editor does not expose when it wants to open and focus an specific line.

**Comment by titoBouzout** [comment]
https://github.com/SublimeTextIssues/Core/issues/313
